extra/twig-extra-bundle/DependencyInjection/TwigExtraExtension.php

```diff
@@ -18,13 +18,36 @@
 use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
 use Twig\Extra\TwigExtraBundle\Extensions;
 
+if (!method_exists(ContainerBuilder::class, 'getAutoconfiguredAttributes')) {
+    /** @internal */
+    trait TwigExtraExtensionTrait
+    {
+        public function load(array $configs, ContainerBuilder $container): void
```

Comment:
why not just adding the return type (-> #4661)?
Especially since the trait is internal.

Reply:
Because the `TwigExtraExtension` it is used in is not `@internal` and adding thus the return type would be a BC break.